src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj

```diff
@@ -24,12 +23,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Using Remove="System.Net.Http" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
-    <PackageReference Include="PolySharp" Version="1.13.2" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
```

Comment:
If global.json has been updated to the dotnet 8 SDK then the SourceLink package is not required anymore.

Reply:
Only package or some other properties can be removed as well?

Reply:
I removed it.